internal/mode/static/leader.go

```diff
@@ -0,0 +1,72 @@
+package static
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	"k8s.io/client-go/rest"
+	"k8s.io/client-go/tools/leaderelection"
+	"k8s.io/client-go/tools/leaderelection/resourcelock"
+	"k8s.io/client-go/tools/record"
+)
+
+const (
+	renewDeadline = 10 * time.Second
```

Comment:
how did we come up with those values?

Reply:
They are the default values used by the core client

Reply:
would it make sense to add a comment that mentions that? this will help any future reader I think